client/state/reader/sites/schema.js

```diff
@@ -0,0 +1,5 @@
+import { sitesSchema } from 'state/sites/schema';
+
+export const readerSitesSchema = Object.assign( {}, sitesSchema );
+
+readerSitesSchema.patternProperties['^\\d+$'].properties.feed_ID = { type: 'number' };
```

Comment:
@blowery, I stumbled upon this file just now. `assign` is shallow, so I think we're mutating `sitesSchema` here, right?

<img width="253" alt="screen shot 2016-06-25 at 17 32 52" src="https://cloud.githubusercontent.com/assets/150562/16357887/142341dc-3afb-11e6-8b68-35c779edb9ef.png">